index.html

```diff
@@ -11,32 +11,201 @@
 			body {
 				font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Helvetica Neue", Arial, sans-serif;
 			}
+
+			#search-field-label {
```

Comment:
this appears unused

Reply:
Ah yea must've missed that on the last pass through! I had a label I decided wasn't necessary in the end.